src/internal/_toString.js

```diff
@@ -18,6 +18,14 @@ module.exports = function _toString(x, seen) {
     return _map(function(k) { return _quote(k) + ': ' + recur(obj[k]); }, keys.slice().sort());
   };
 
+  //  Object.prototype.toString.call(x) does not behave correctly in
+  //  versions of PhantomJS prior to 2.0 when x is null or undefined.
+  if (x === null) {
+    return 'null';
+  } else if (x === void 0) {
+    return 'undefined';
+  }
+
```

Comment:
This is a workaround for ariya/phantomjs#11722. We should be able to revert this change once Travis gives us the option of running PhantomJS 2.0 rather than 1.9.